libnopegl/src/node_media.c

```diff
@@ -149,12 +149,13 @@ static void callback_nopemd_log(void *arg, int level, const char *filename, int
 
     /* handle the case where the line doesn't fit the stack buffer */
     if (len >= sizeof(logline)) {
-        logbuf = ngli_malloc(len + 1);
+        const size_t logbuf_len = (size_t)len + 1;
```

Comment:
Can you name the variable `logbuf_size` or just `n`? Semantically I like to use `len` as being explicitly about string length, meaning not accounting for the trailing nul, as opposed to `size`

Reply:
Done, also did the same for the log commit.